api/v1alpha1/bundle_types.go

```diff
@@ -154,15 +154,16 @@ type BundleStatus struct {
 	ContentURL         string             `json:"contentURL,omitempty"`
 }
 
-//+kubebuilder:object:root=true
-//+kubebuilder:resource:scope=Cluster
-//+kubebuilder:subresource:status
-//+kubebuilder:printcolumn:name=Type,type=string,JSONPath=`.spec.source.type`
-//+kubebuilder:printcolumn:name=Phase,type=string,JSONPath=`.status.phase`
-//+kubebuilder:printcolumn:name=Age,type=date,JSONPath=`.metadata.creationTimestamp`
-//+kubebuilder:printcolumn:name=Provisioner,type=string,JSONPath=`.spec.provisionerClassName`,priority=1
-//+kubebuilder:printcolumn:name=Resolved Source,type=string,JSONPath=`.status.resolvedSource`,priority=1
-
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Cluster
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name=Type,type=string,JSONPath=`.spec.source.type`
+// +kubebuilder:printcolumn:name=Phase,type=string,JSONPath=`.status.phase`
+// +kubebuilder:printcolumn:name=Age,type=date,JSONPath=`.metadata.creationTimestamp`
+// +kubebuilder:printcolumn:name=Provisioner,type=string,JSONPath=`.spec.provisionerClassName`,priority=1
+// +kubebuilder:printcolumn:name=Resolved Source,type=string,JSONPath=`.status.resolvedSource`,priority=1
+// +kubebuilder:storageversion
```

Comment:
This was intentional. Wanted to finish the entire refactoring of controllers so that we have all the code in, and towards the end, make v1alpha2 stored as well as cleanup the references and additional controllers (like crd validation) which we have been using because of bundle APIs and v1alpha1 design. Was planning to open refactor PRs in chunks so that its easier to review, and make the switch in the end.

Reply:
I am not sure to follow why it is useful but as long as the latest version becomes eventually the stored one it is fine with me.

Reply:
This was to not break the current e2e tests or the existing code as well as make the review process easier during refactoring. Yeah, once everything gets in, will clean alpha1, bundle API references and switch the storage version.